styles/styles.css

```diff
@@ -4,9 +4,10 @@
     font-family: "Roboto", sans-serif;
     box-sizing: border-box;
 }
+
 .container {
     background-color: #ffffff;
-    width: 1440px;
+    max-width: 1440px;
     height: 570px;
```

Comment:
add margin parameter please, now you have some troubles in screen width bigger than 1440px
![image](https://user-images.githubusercontent.com/79144475/197388628-9e5fd692-5e11-4a69-93fb-d68cfd2b1b08.png)

if you add auto parameter for margin sides - you fix it easy
![image](https://user-images.githubusercontent.com/79144475/197388703-6d152c9c-51ad-47d9-954f-d305071f8a31.png)